std/process.d

```diff
@@ -1266,6 +1266,61 @@ final class Pid
         return _processID;
     }
 
+    version (Windows)
+    {
+        /**
+        A simple factory that returns a $(LREF Pid) handle when provided with a
```

Comment:
That would be new to me.
`LREF` = local reference (within the module), only `XREF` was deprecated.